tests/hooks/pager.spec.ts

```diff
@@ -104,6 +150,48 @@ describe('Hook `usePager`', () => {
         checkPage({ callNumber: 1, pageNumber: 2, searchParams });
     });
 
+    test('method `loadPrevious`', async () => {
+        const data = success({
+            id: 'fakeID',
+            resourceType: 'type',
+            link: [
+                {
+                    relation: 'first',
+                    url: '/EpisodeOfCare?page=1',
+                },
+                {
+                    relation: 'next',
+                    url: '/EpisodeOfCare?page=3',
+                },
+                {
+                    relation: 'self',
+                    url: '/EpisodeOfCare?page=2',
+                },
+                {
+                    relation: 'previous',
+                    url: '/EpisodeOfCare?page=1',
+                },
+            ],
+        });
+
+        const searchParams = { a: 1, b: 2 };
+
+        (<jest.Mock>useService).mockImplementation(() => [data]);
+
+        const { result } = renderHook(() => usePager<Bundle<any>>(resourceType, resourcesOnPage, searchParams));
+        const {
+            current: [, { loadPrevious }],
+        } = result;
+
+        checkPage({ callNumber: 0, pageNumber: 1, searchParams });
+
+        act(() => {
+            loadPrevious();
+        });
+
+        checkPage({ callNumber: 1, pageNumber: 0, searchParams });
```

Comment:
And after navigating to the previous page it should become `1`

```suggestion
        checkPage({ callNumber: 1, pageNumber: 1, searchParams });
```